packages/target-tools/src/utils.js

```diff
@@ -281,3 +281,38 @@ export function executeTelemetries(deliveryRequest, telemetryEntries) {
     }
   };
 }
+
+export function isExecutePageLoad(request) {
+  return !!request.execute && !!request.execute.pageLoad;
+}
+
+export function executeMboxCount(request) {
+  return (
```

Comment:
Here and below, we can probably drop the redundant parentheses
```suggestion
  return 
```

Reply:
Prettier really wants is there for some reason :/